Update identity-providers.mdx (#203)
gobeyondidentity/customer-docs: docs/access-control/identity-providers/identity-providers.mdx

displayed_sidebar: mainSidebar
---

> The Identity Providers section in Beyond Identity allows IT administrators to integrate existing Identity Providers (IDPs) into their authentication workflow. This configuration ensures that users must verify their identity with an external IDP before they can enroll in a Beyond Identity Credential. By leveraging your existing IDP for this step, Beyond Identity enhances your passwordless authentication deployment with an added layer of security and user verification.

Reviewer: ```suggestion
The Identity Providers (IdPs) functionality in Beyond Identity is for IT administrators to integrate existing Identity Providers into their authentication workflow. This configuration ensures that users verify their identity with an external IdP before they can enroll in a Beyond Identity credential. By leveraging your existing IdP for this step, Beyond Identity enhances your passwordless authentication deployment with an added layer of security and user verification.
```
